src/main/webapp/index.html

```diff
@@ -45,6 +45,10 @@
         <div class="modal-header"> 
           <div class="modal-title">Results</div>
           <button id="modal-backarrow"></button>
+          <div class="button-group">
+            <button> Recent </button>
```

Comment:
Remove spaces before and after Recent / Relevant as per html style guidelines!

Reply:
Dangit I remebered there was something I had to do with it I just forgot what it was.